src/components/SlateEditor/plugins/toolbar/ToolbarLanguageOptions.tsx

```diff
@@ -35,9 +38,21 @@ export const ToolbarLanguageOptions = () => {
   const { t } = useTranslation();
   const editor = useSlate();
   const currentLanguage = useSlateSelector(getCurrentLanguage);
+  const [open, setOpen] = useState(false);
+  const [scrollDropdown, setScrollDropdown] = useState(false);
+
+  const handleOpenDropdown = () => {
+    setOpen(true);
+    if (window.innerHeight - BOTTOM_TOOLBAR_HEIGHT < languages.length * LANGUAGE_BUTTON_HEIGHT) {
+      setScrollDropdown(true);
+    } else {
+      setScrollDropdown(false);
+    }
+  };
```

Comment:
Hvorfor trenger vi dette?

Reply:
Vanskelig å svare på, sida hva 'dette' er, ikke er helt tydelig for meg. Men kort oppsummert:
- Jeg valgte å løse scrolle-oppførselen på dropdown ved å sette data-attributt, og sida den er avhengig av `window.innerHeight` ga det mest mening å ha det som en state, i tilfelle vindu resizes.
- Jeg fikk ikke tak i noen state for når dropdown var åpen eller ikke uten å lage og passe inn den staten selv, så det er angitt her, og dermed slått sammen med hvor vidt data-attributtet settes eller ikke
- Jeg syns det var ryddigere å legge koden utafor markupen, i en egen funksjon
- if-sjekken er resultat av trial & error, og basert på at jeg må sjekke hvor mye tilgjengelig plass som er igjen i vinduet etter at høyde på menylinja i bunnen er borte, og basert på hvor mange knapper som er i nedtrekksmenyen (slik at utvidelse med flere språk ikke knekker denne funksjonaliteten)

Jeg er åpen for forslag om hvordan bedre løse det, og hvis jeg misforstod hva du mente og derfor ikke svarte på spørsmålet, hyl ut :)